pkg/services/business_case.go

```diff
@@ -155,7 +155,7 @@ func NewAuthorizeUpdateBusinessCase(logger *zap.Logger) func(
 			return false, nil
 		}
 		// If intake is owned by user, authorize
```

Comment:
Oops, I think this was my copy paste error
```suggestion
		// If business case is owned by user, authorize
```